tests/compile/test_backend.py

```diff
@@ -1,42 +1,15 @@
 import pytest
 
-from myia.compile.backends import (
-    LoadingError,
-    UnknownBackend,
-    load_backend,
-    parse_default,
-)
+from myia.compile.backends import LoadingError, UnknownBackend, load_backend
 from myia.operations import (
     array_reduce,
     array_to_scalar,
     reshape,
     scalar_add,
     scalar_to_array,
 )
-
-from ..common import AN, MA
-from ..multitest import mt, run, run_gpu
-
-
-def test_default_backend():
-    import os
-
-    before = os.environ.get("MYIA_BACKEND", None)
-    try:
-        os.environ["MYIA_BACKEND"] = "pytorch"
-        assert parse_default() == ("pytorch", {})
-
-        os.environ["MYIA_BACKEND"] = "pytorch?target=cpu"
-        assert parse_default() == ("pytorch", {"target": "cpu"})
-
-        os.environ["MYIA_BACKEND"] = "relay?target=cpu&device_id=0"
-        assert parse_default() == ("relay", {"target": "cpu", "device_id": "0"})
-    finally:
-        # Make sure we don't switch the default for other tests.
-        if before is None:
-            del os.environ["MYIA_BACKEND"]
-        else:
-            os.environ["MYIA_BACKEND"] = before
```

Comment:
I think that this should stay here.  This is a test of the string parser and doesn't actually require importing any backend.